app/services/geek_jokes.js

```diff
@@ -0,0 +1,17 @@
+const axios = require('axios');
+const config = require('../../config');
+
+const getRandomJoke = async () => {
+  const options = {
+    params: {
+      format: 'json'
+    }
+  };
+  console.info('Getting random joke...');
+  return axios.default.get(config.common.geek_joke_api.url, options)
+    .then(response => response.data);
```

Comment:
you should use the custom Wolox logger (app.logger.js) in the catch with the custom error (app/erros.js)

Reply:
`console.log` changed to `logger.info`